embuilder.py

```diff
@@ -128,6 +128,7 @@ def main():
                                    formatter_class=argparse.RawDescriptionHelpFormatter,
                                    epilog=get_help())
   parser.add_argument('--lto', action='store_true', help='build bitcode object for LTO')
+  parser.add_argument('--lto=thin', action='store_true', help='build bitcode object for ThinLTO')
```

Comment:
You think you can do `dest='lto', action='store_const', const='thin'` ..

Reply:
this means also doing `const='full'` in `--lto` ?

Reply:
Its up to you.     You don't have to do that.    You could just do something like this:

```
if args.lto == 'thin'
   settings.LTO = "thin"	
else args.lto:
   settings.LTO = "full"
```

I don't think this matter much which way you do it.

Reply:
ok, I did the other way because the code was shorter. The tests failure look unrelated.